tests/test_reporters.py

```diff
@@ -0,0 +1,115 @@
+import unittest
```

Comment:
Stuff still needed

- [ ] Test that functions with all three problems don't produce three doppel errors (e.g. that early stopping in `_check_function_arg_names()1 works
- [] Change `_check_function_arg_names()` to `check_function_args()`
- test that packages with identical sets of functions return no errors
- test that packages with NO packages return no errors